api/events/lambda_function.py

```diff
@@ -99,10 +101,31 @@ def lambda_handler(event, _):
         if "resource_arn" in item:
             print("Storing media service event.")
             print(item)
-            CLOUDWATCH_EVENTS_TABLE_NAME.put_item(Item=item)
+            CLOUDWATCH_EVENTS_TABLE.put_item(Item=item)
         else:
             print("Skipping this event. " + item["type"])
-
     except ClientError as error:
         print(error)
     return True
+
+
+def get_pipeline_state(event):
```

Comment:
When we call this function we already know we're not a multiplex, cause we check for that. So we probably only need to pass in the ARN to this function, and not the whole event. We don't need to check again that we are a multiplex before we run the DB query.

Reply:
True. Now that we are to support multiplex this will be updated shortly